addons/tm_bear_fieldmanual_rpg7v2/CfgVehicles.hpp

```diff
@@ -0,0 +1,18 @@
+class CfgVehicles
+{
+	class Man;
+	class CAManBase: Man 
+	{
+		class ACE_SelfActions
+		{
+			class bear_fieldmanual_rpg7v2
+			{
+				displayName = "RPG-7V2 Field Manual";
+				condition = "(secondaryWeapon player) isEqualTo 'rhs_weap_rpg7'";
+				statement = "0 = 0 spawn {findDisplay 46 createDisplay 'RscBear_fieldManual_rpg7v2';};";
```

Comment:
This is still wonky looking, should be doable to trim down to ``spawn {findDisplay 46 createDisplay 'RscBear_fieldManual_rpg7v2';};``